Frameworks/CoreText/CTLine.mm

```diff
@@ -16,16 +16,48 @@
 
 #import <CoreText/CTLine.h>
 #import <StubReturn.h>
-#import "NSStringInternal.h"
 #import "CoreTextInternal.h"
 #import "CGContextInternal.h"
 #import "DWriteWrapper_CoreText.h"
 #import <CoreText/CTTypesetter.h>
 
 #import <algorithm>
 #import <numeric>
+#import <cwchar>
 #import <vector>
 
+static CFStringRef __CTCreateReversedString(CFStringRef string) {
```

Comment:
UnitTests

---
In reply to: [111007038](https://github.com/Microsoft/WinObjC/pull/2479#discussion_r111007038) [](ancestors = 111007038)